### DISCUSSION TITLE
CREALITY_CR4NTxxC10 E3 Free-runs

### DISCUSSION THREAD

**Discussion by Mystictoad** [mriscoc/Ender3V2S1#1272] [Q&A]
CREALITY_CR4NTxxC10 E3 Free-runs
 Which .BIN file of Professional Firmware for 3D Printers do I use. It state it supports it.
 New Year 2024 (20240122) What's new in this release Fix CREALITY_CR4NTxxC10 E3 Free-runs
 The board has a STM32F401RCT6 chip
 Thank you for your help

**Comment by mriscoc** [comment]
You must build a custom version yourself using the Configurator or request a binary for a small fee.

**Comment by Mystictoad** [comment]
How much would it cost for the binary thank you for replying 






On Tuesday, March 12, 2024 at 11:06:25 PM MDT, Miguel Risco-Castillo ***@***.***> wrote: 







You must build a custom version yourself using the Configurator or request a binary for a small fee.

**Reply by mriscoc** [reply]
$5 for Enhanced support subscribers and $10 for Simple support subscribers in Patreon.

**Comment by Mystictoad** [comment]
Request a onetime firmware build using the settings of the current version of the firmware and Configurator for an additional value of US$ 10, please use Paypal to send the US$10 or equivalent
You donated $10.00 USD ***@***.***

Thank you for using PayPal

Donation Details



Date
March 15, 2024
Transaction ID
38B27536PC8547156
Purpose

Donation to
***@***.***
Donation from
***@***.***
Donation amount
$10.00 USD
Total amount of this transaction    $10.00 USD


Funding Sources Used (Total)

PayPal Credit    $10.00 USD

P.S. I have BLT and use UBL THANK YOU






On Thursday, March 14, 2024 at 09:02:13 PM MDT, Miguel Risco-Castillo ***@***.***> wrote: 







$5 for Enhanced support subscribers and $10 for Simple support subscribers in Patreon.

**Reply by mriscoc** [reply]
Hi, I responded using your Paypal registered email, thank you!
